### ISSUE TITLE
TagToRDF on document containing a PackageComment fails with InvalidSpdxTagFileException

### ISSUE THREAD

**Issue by ttgurney** [spdx/tools#24]
Attempting to convert this file from tag to RDF:
https://gist.github.com/ttgurney/54abaae84f5274d6b739

```
$ java -jar spdx-tools-2.0.1-SNAPSHOT-jar-with-dependencies.jar TagToRDF yocto-spdx.tag yocto-spdx.rdf
Error creating SPDX Analysis: org.spdx.tag.InvalidSpdxTagFileException: Expecting a definition of a file, package, license information, or document property at PackageComment:
```

Removing the `PackageComment` line from the document allows it to convert successfully.

UPDATE: Looks like the issue is here:
https://github.com/spdx/tools/blob/master/src/org/spdx/tag/SpdxTagValueConstants.properties#L70
But changing the `PackageComment:` to `Comment:` in the above tag file still produced an error on attempting to convert.

**Comment by romeara** [comment]
I'm putting together a pull request for this. Still outputs 
```
The following verifications failed for the SpdxDocument:
	Invalid date format: Unparseable date: "2015-06-16 12:09:53.533268"
	Missing required license text for LicenseRef-CCPL
	Missing required license text for LicenseRef-ZPL
	Missing required license text for LicenseRef-MIT-style
	Missing required license text for LicenseRef-MIT-possibility
	Missing required license text for LicenseRef-Python
```

but appears to output an RDF file, despite these reported issues
	Missing required license text for LicenseRef-MPL

**Comment by ttgurney** [comment]
Thanks! The remaining failed validations are known bugs on my end.

**Comment by romeara** [comment]
@goneall Merged in the patch - this should be resolved in the next release

**Comment by ybronshteyn** [comment]
Fixed in 2.0.2 release
